src/Sarif/Schemata/sarif-2.1.0-rtm.6.json

```diff
@@ -1709,24 +1709,28 @@
         "startLine": {
           "description": "The line number of the first character in the region.",
           "type": "integer",
+          "default": 0,
           "minimum": 1
```

Comment:
>"minimum": 1 [](http://example.com/codeflow?start=10&length=12)

set all minimums to default and then let's please make every int value that has a default in the schema be a nullable item.

of course, we are skipping region for now. I wonder if we should build a special region deserializer! :)